graphql_compiler/tests/test_emit_output.py

```diff
@@ -289,3 +294,239 @@ def test_datetime_output_representation(self):
 
         received_match = emit_gremlin.emit_code_from_ir(ir_blocks, None)
         compare_gremlin(self, expected_gremlin, received_match)
+
+
+class EmitCypherTests(unittest.TestCase):
+    def setUp(self):
+        """Disable max diff limits for all tests."""
+        self.maxDiff = None
+
+    def test_simple_immediate_output(self):
+        # corresponds to:
+        # graphql_string = '''{
+        #     Foo {
+        #         name @output(out_name: "foo_name")
+        #     }
+        # }'''
+
+        base_location = Location(('Foo',))
+        base_name_location = base_location.navigate_to_field('name')
+        base_location_info = LocationInfo(
+            parent_location=None,
+            type=GraphQLObjectType(name='Foo', fields={'name': GraphQLString}),
+            coerced_from_type=None,
+            optional_scopes_depth=0,
+            recursive_scopes_depth=0,
+            is_within_fold=False,
+        )
+
+        ir_blocks = [
+            QueryRoot({'Foo'}),
+            MarkLocation(base_location),
+            GlobalOperationsStart(),  # necessary for Cypher. Filter/output blocks come after this.
```

Comment:
It should exist for all backends -- please add it if it's missing. `ConstructResult` is also a global operation.

Reply:
Will modify in a separate PR so this PR is easier to review (this one will cover only the new Cypher tests)